src/zeep/transports.py

```diff
@@ -5,16 +5,19 @@
 import requests
 
 from six.moves.urllib.parse import urlparse
+from zeep import __version__
 from zeep.cache import SqliteCache
-from zeep.utils import NotSet, get_version
 from zeep.wsdl.utils import etree_to_string
 
 
+ZEEP_USERAGENT = 'Zeep/%s (www.python-zeep.org)' % __version__
+
+
 class Transport(object):
     supports_async = False
 
-    def __init__(self, cache=NotSet, timeout=300, operation_timeout=None,
-                 verify=True, http_auth=None):
+    def __init__(self, cache=SqliteCache(), timeout=300, operation_timeout=None,
+                 verify=True, http_auth=None, http_headers={}):
```

Comment:
This will init the SqliteCache during startup/import. This isn't something we want since sqlite is not always available and we want users to let them choose other cache backends in that case.